lightwood/data/timeseries_transform.py

```diff
@@ -75,7 +75,7 @@ def transform_timeseries(
                     raise Exception(
                         f"Partition is not valid, faulty group {group}. Please make sure you group by a set of columns that ensures unique measurements for each grouping through time.")  # noqa
 
-                index = pd.to_datetime(subset[oby_col], unit='s')
+                index = pd.to_datetime(subset[oby_col], unit='s', utc=True)
```

Comment:
UTC is the only time :D 

There might be an edge case where leap seconds don't get properly handled (https://en.wikipedia.org/wiki/Leap_second) but this is very, very unlikely to cause trouble; it did cause trouble to me when working with astronomical data so it might be worth to keep a mental post-it about it.